src/main/java/org/example/game/Sessions.java

```diff
@@ -0,0 +1,55 @@
+package org.example.game;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import java.util.HashMap;
+import java.util.LinkedHashMap;
+import java.util.Map;
+import java.util.UUID;
+
+public class Sessions {
+    private Map<UUID, Map<UUID, Game>> sessions = new HashMap<>();
+
+    public void addGameToSession(UUID sessionId, UUID gameId, Game game) {
+        // Get or create map for session
+        sessions.computeIfAbsent(sessionId, k -> new LinkedHashMap<>());
```

Comment:
I assume you are using LinkedHashMAp to be able to get the last game in a session. But why do you need that? In what user flow you will know the session ID, but you will `lose` your game ID?